api/net/Areas/Editor/Controllers/StorageController.cs

```diff
@@ -29,17 +29,17 @@ namespace TNO.API.Areas.Editor.Controllers;
 public class StorageController : ControllerBase
 {
     #region Variables
-    private readonly StorageConfig _config;
+    private readonly StorageOptions _options;
```

Comment:
I've seen different conventions for naming these files.  I'm not sure which I prefer.